src/main/java/org/kitteh/irc/client/library/util/Mask.java

```diff
@@ -56,22 +174,70 @@ private Mask(@Nonnull String string) {
      */
     @Nonnull
     public String asString() {
-        return this.string;
+        final StringBuilder sb = new StringBuilder();
+        sb.append(this.getNick().orElse(WILDCARD));
+        sb.append('!');
+        sb.append(this.getUser().orElse(WILDCARD));
+        sb.append('@');
+        sb.append(this.getHost().orElse(WILDCARD));
+        return sb.toString();
+    }
+
+    /**
+     * Gets a collection of users that match this mask in the provided channel.
```

Comment:
You should note about the shortcomings of this method and how it's not the same as the set of users the IRCd would ban for a `+b` set with the exact same mask.